src/main/telemetry/jetiexbus.c

```diff
@@ -155,16 +186,16 @@ uint8_t calcCRC8(uint8_t *pt, uint8_t msgLen)
   -----------------------------------------------
    Jeti Ex Bus Telemetry
   -----------------------------------------------
-*/
+ */
```

Comment:
That's an odd way of formatting this comment - normally an aligned `*` is used at the beginning of every comment line

Reply:
Is corrected.